### ISSUE TITLE
Test suite failure on Stackage build machine

### ISSUE THREAD

**Issue by snoyberg** [jtdaugherty/brick#375]
I believe this was introduced with the 0.71 bump. I can include fuller logs if useful, but I believe the relevant bit is:

```
brick-tests: setupTerm: Couldn't look up terminfo entry "unknown"
```

**Comment by jtdaugherty** [comment]
Thanks for reporting this! This is now fixed in 0.71.1.

**Comment by snoyberg** [comment]
Thanks for the quick response!
